src/Sulu/Bundle/ContactBundle/Controller/AbstractContactController.php

```diff
@@ -132,6 +131,11 @@ protected function addNewContactRelations($contact, Request $request)
                 $this->addTag($contact, $tag);
             }
         }
+
+        // process details
+        if (!is_null($request->get('bankAccounts'))) {
```

Comment:
Please don't start using `is_null`, we almost (?) always used `=== null` until now, which is also a little bit faster.